### ISSUE TITLE
[Question] Sorry how does dependenpy, archan, and meerkat related to one another?

### ISSUE THREAD

**Issue by simkimsia** [pawamoy/dependenpy#47]
I was googling about design structure matrices and python when I stumble upon this project.

I like to know how the 3 projects are related to one another and how to use them to improve the architecture of code written in a Django app. Assume LTS 2.2 and above

**Comment by pawamoy** [comment]
Hello!

For the story, I first wrote dependenpy, which computes the dependencies between modules of several packages, and then archan, which uses these data to assess some architecture criteria. Django Meerkat was supposed to be a all-in-one, supporting plugins such as archan+dependenpy to show the analysis results directly in the admin panel, among other things.

The development of these projects was motivated by security research a few years ago. I cannot guarantee the validity of the analysis :smile:. It was a bit experimental if you see what I mean. 

Meerkat still has some nice code, unfortunately I don't have the time to maintain it. I didn't even write documentation for it, what a shame...
I'd like to give a bit of love to Archan, but my time is limited (I'm not sure it still works out of the box).

Dependenpy however should work properly, but will only give you the data, not the criteria analysis.

I hope I answered your question, don't hesitate to ask again!

(Closing since no actionable item).
